README.md

```diff
@@ -30,27 +30,27 @@ The minimum required version is 2020.1**.
 <!-- To install doctoc run `npm install -g doctoc`, to use it run `doctoc <this-file-path> -->
 # Table of Contents
 
-- [Git Machete IntelliJ Plugin](#git-machete-intellij-plugin)
```

Comment:
This changes look like some glitch in GH, they shouldn't be here IIUC (they're already merged to `develop`)